admin-part/src/main/kotlin/CoverageTools.kt

```diff
@@ -73,9 +73,10 @@ internal fun CoverContext.calculateBundleMethods(
 
 internal fun Map<TypedTest, BundleCounter>.methodsCoveredByTest(
     context: CoverContext,
-    cache: AtomicCache<Int, MethodsCoveredByTest>?
+    cache: AtomicCache<TypedTest, MethodsCoveredByTest>?,
+    finalizedTests: Sequence<TypedTest>?
```

Comment:
Can you make it non-nullable?

Reply:
no

Reply:
If you pass an empty sequence in the parameter, there is no need to check for null